ipn/ipnstate/ipnstate.go

```diff
@@ -196,6 +196,10 @@ type PeerStatusLite struct {
 	LastHandshake time.Time
 	// NodeKey is this peer's public node key.
 	NodeKey key.NodePublic
+	// HandshakeAttempts is how many attempts at completing the
```

Comment:
Maybe "unsuccessful attempts" since presumably if it succeeds the first time it will never become nonzero?